protect-function-app/README.md

```diff
@@ -3,7 +3,7 @@ page_type: sample
 languages:
 - nodejs
 name: Node.js Azure Function protecting HTTP trigger function with Easy Auth and access token scope validation.
-description: This Node.js Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on docs.microsoft.com.
+description: This Node.js Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on learn.microsoft.com.
```

Comment:
Maybe we could add a link here?  https://learn.microsoft.com/entra/identity-platform/ maybe?